stories/settings.stories.js

```diff
@@ -91,8 +92,11 @@ storiesOf( 'Settings', module )
 		global._googlesitekitLegacyData.canAdsRun = true;
 		global._googlesitekitLegacyData.modules.analytics.setupComplete = false;
 		global._googlesitekitLegacyData.modules.adsense.active = false;
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( CORE_MODULES ).registerModule( 'adsense' )
```

Comment:
Missing semicolon here, this makes linting fail on Travis-CI.

Reply:
Addressed.